ntoskrnl/ps/debug.c

```diff
@@ -59,6 +59,7 @@ PspDumpThreads(BOOLEAN IncludeSystem)
                    Thread->Tcb.State == Standby ||
                    Thread->Tcb.State == Waiting)
                 {
+#ifdef _M_IX86
                     ULONG i = 0;
```

Comment:
Not a blocker for this change, but I wonder why we don't use RtlWalkFrameChain for this.

Reply:
Yeah that is what we should use eventually, but for now I think it's fine to comment it out to get further with stuff.